src/components/common/productPickerModal.tsx

```diff
@@ -2,7 +2,7 @@ import { Button, Checkbox, Dialog, DialogActions, DialogContent, DialogTitle } f
 import { getDictionary, getProductsPaged } from 'api/admin';
 
 import { GetProductsPagedRequest, common_Category, common_Product } from 'api/proto-http/admin';
-import { initialFilter } from 'components/managers/products/listProducts/filterComponents/initialFilterStates';
+import { initialFilter } from 'components/managers/products/listProducts/filterProdcuts/initialFilterStates';
```

Comment:
typo 'filterProdcuts'

Reply:
and why do we import to common something from /products ? common is a reusable methods without any dependency with concrete manager